src/elektra/redshift.ini

```diff
@@ -0,0 +1,308 @@
+; Elektra specification file for redshift@490ba2aae9cfee097a88b6e2be98aeb1ce990050 (= redshift v1.12 + some small commits)
+;
+; A specification file describes which configuration settings an application supports, their defaults and their associated CLI options. It also "(...) defines how the configuration settings should be validated" (source: https://www.libelektra.org/tutorials/validate-configuration)
+; For details see: https://www.libelektra.org/tutorials/writing-a-specification-for-your-configuration)
+
+; Redshift uses no environment variables.
+; Note: When renaming enum values, make sure to also update any code that uses them.
+; (e.g. the string values of enum [provider/location] are used in options.c).
+
+[]
+mountpoint = redshift.ecf
+
+[mode]
+type = enum
+check/enum = #4
+check/enum/#0 = continual
+check/enum/#1 = print
```

Comment:
This prints the location in a format that is not suitable to be used in the config:

```
kdb set user:/sw/redshift/#0/current/provider/location/manual/lat "47.31 N"                                                                                             
#> Sorry, module type issued the error C03200:
#> Validation Semantic: The type 'float' failed to match for 'user:/sw/redshift/#0/current/provider/location/manual/lat' with string '47.31 N'
```

Reply:
Well spotted, thanks! I changed the output format to be compatible with the config.